src/main/controllers/UserResultsController.ts

```diff
@@ -42,9 +42,10 @@ export class UserResultsController extends RootController {
       }
 
       return super.post(req, res, 'manage-user', {
```

Comment:
I think there is a private method called postError(0 in this class. You can just call the method directly